controller/src/test/java/io/pravega/controller/task/Stream/StreamTransactionMetadataTasksTest.java

```diff
@@ -208,8 +210,8 @@ public void commitAbortTests() {
                 abortWriter);
 
         // Create ControllerService.
-        consumer = new ControllerService(streamStore, hostStore, streamMetadataTasks, txnTasks,
-                segmentHelperMock, executor, null);
+        consumer = new ControllerService(streamStore, hostStore, streamMetadataTasks, txnTasks, segmentHelperMock,
```

Comment:
instead of changing all the tests here with ControllerService constructor and supplying `new StreamMetrics(), new TransactionMetrics()` , we could have a constructor in controllerService that does just that internally.

Reply:
Done.